app/configurator/configurator-state/context.tsx

```diff
@@ -271,7 +271,10 @@ export const ConfiguratorStateProviderInternal = (
           } else if (query.edit && typeof query.edit === "string") {
             newChartState = await initChartStateFromChartEdit(
               client,
-              query.edit
+              query.edit,
+              typeof query.overrideState === "string"
+                ? query.overrideState
```

Comment:
Nit: I think there is no need for "override", maybe simply "state" or "configuratorState"